src/IO/HDF5/HDF5IOHandler.cpp

```diff
@@ -579,24 +664,50 @@ void HDF5IOHandlerImpl::createDataset(
 
         H5Pset_fill_time(datasetCreationProperty, H5D_FILL_TIME_NEVER);
 
-        if (num_elements != 0u && m_chunks != "none")
+        if (num_elements != 0u && chunking.has_value())
         {
-            //! @todo add per dataset chunk control from JSON config
-
-            // get chunking dimensions
-            std::vector<hsize_t> chunk_dims =
-                getOptimalChunkDims(dims, toBytes(d));
-
             //! @todo allow overwrite with user-provided chunk size
             // for( auto const& val : parameters.chunkSize )
             //    chunk_dims.push_back(static_cast< hsize_t >(val));
```

Comment:
@franzpoeschel can this todo now be removed?

Reply:
yep, this todo is now fixed